tests/test_torch_agent.py

```diff
@@ -263,26 +263,29 @@ def test_batchify(self):
         """
         Make sure the batchify function sets up the right fields.
         """
-        agent = get_agent(rank_candidates=True)
+        agent = get_agent(rank_candidates=True, timestep_field='timestep')
```

Comment:
could we alternate messages with and without timesteps? to test both cases?

Reply:
Added a test case in line 933 of the test file